tests/screenplay.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-undef*/
+/* eslint-disable no-unused-expressions*/
+/* eslint-disable func-names */
+const chai = require('chai');
+const moderator = require('../src/moderator');
+
+const expect = chai.expect;
+
+describe('Screenplay', () => {
+    describe('fromGame()', () => {
```

Comment:
I would add tests for each specific filter scenario.

`witch` should have all level
`cult_leader` should have deck level
`villager` should have game

Don't you think is necessary?